docs/stack/get-started/tutorials/stack-spring.md

```diff
@@ -578,15 +578,15 @@ Iterable<Person> byHomeLoc(//
 
 Refreshing the Swagger US, we should now see the `byHomeLoc` endpoint. Let’s see which of the Avengers live within 10 miles of Suffolk Park Pub in South Wales, Australia... hmmm.
 
-![SwaggerUI](./stack-spring/images/017_stack_spring.png "SwaggerUI")
+![SwaggerUI](./images/017_stack_spring.png "SwaggerUI")
 
 Executing the request, we get the record for Chris Hemsworth:
 
-![SwaggerUI](./stack-spring/images/018_stack_spring.png "SwaggerUI")
+![SwaggerUI](./images/018_stack_spring.png "SwaggerUI")
 
 and in Redis Insight we can see the backing RediSearch query:
 
-![RedisInsight](./stack-spring/images/019_stack_spring.png "RedisInsight Profiler")
+![RedisInsight](./images/019_stack_spring.png "RedisInsight Profiler")
```

Comment:
Not sure this will fix the issue, I'm doing the same thing in the dotnet tutorial but on the website the links don't work, probably because the resources aren't being pulled into to anywhere we know about. Perphaps the top-level images directory is where everything needs to be moved to.

Reply:
Dang it. I'll guess I'll wait until the Hugo Gods tell us what's what!